src/test/java/com/yahoo/bullet/storm/FilterBoltTest.java

```diff
@@ -624,17 +677,18 @@ public void testGroupAllCount() {
         GroupData actual = SerializerDeserializer.fromBytes(getRawPayloadOfNthTuple(1));
         BulletRecord expected = RecordBox.get().add("cnt", 10L).getRecord();
 
-        Assert.assertTrue(isEqual(actual, expected));
+        Assert.assertEquals(actual.getMetricsAsBulletRecord(provider), expected);
+        //Assert.assertTrue(isEqual(actual, expected));
```

Comment:
?

Reply:
Idk why I commented this one out. It passes.